config.rb

```diff
@@ -76,6 +76,8 @@ def self.test?
   override :versioning, false, bool
   optional :hetzner_user, string, clear: true
   optional :hetzner_password, string, clear: true
+  override :hetzner_hostname, string
```

Comment:
Out of curiosity....why is the hostname important?

Reply:
(we talked it over, it's a way to avoid using the API to get the dial-able name for SSH, but I suggested loading the hostname out of the API to reduce a configuration variable is a good economy)